src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/jsx-props-no-multi-spaces */
+import _ from 'underscore';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {FlatList} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import MenuItem from '../../../components/MenuItem';
+import compose from '../../../libs/compose';
+import withLocalize from '../../../components/withLocalize';
+import ONYXKEYS from '../../../ONYXKEYS';
+import {
+    Bank,
+    CreditCard,
+    PayPal,
+    Plus,
+} from '../../../components/Icon/Expensicons';
+
+const propTypes = {
+    // What to do when a menu item is pressed
```

Comment:
Small code style detail but all the comments for propTypes should have this format:

```javascript
/** Prop description */
```